Frontend-Angular/data-polish-frontend/angular.json

```diff
@@ -102,5 +102,8 @@
         }
       }
     }
+  },
+  "cli": {
```

Comment:
Why we need this

Reply:
That was from the auto create component command. I dont know why it's in there. Must be needed for it to work?